dts/xtensa/intel/intel_adsp_ace15_mtpm.dtsi

```diff
@@ -121,9 +121,38 @@
 			reg = <0x28800 0x1000>;
 		};
 
-		win: win@70200 {
-			compatible = "intel,cavs-win";
-			reg = <0x70200 0x30>;
+		mem_window0: mem_window@70200 {
+			compatible = "intel,adsp-mem-window";
+			reg = <0x70200 0x8>;
+			size = <0x2000>;
+			offset = <0x4000>;
+			memory = <&sram0>;
+			initialize;
+			status = "okay";
+		};
+		mem_window1: mem_window@70208 {
+			compatible = "intel,adsp-mem-window";
+			reg = <0x70208 0x8>;
+			size = <0x2000>;
+			offset = <0x6000>;
+			memory = <&sram0>;
+		};
+
+		mem_window2: mem_window@70210 {
+			compatible = "intel,adsp-mem-window";
+			reg = <0x70210 0x8>;
+			memory = <&sram0>;
+			size = <0x2000>;
+			offset = <0x8000>;
```

Comment:
For this we need some work on application (SOF) side to define DT overlays, as we two different schemes for the semantics of windows 2 and 3. We now use Kconfig overlays when building for different schemes with west. This mostly affects the cavs25 Intel target. For other ADSP targets we have in Zephyr, the expectation is that all apps will use the same definition (so the values in upstreadm Zephyr DT will be used by everybody).